package-build/package-build.el

```diff
@@ -44,6 +44,7 @@
 (require 'package)
 (require 'lisp-mnt)
 (require 'json)
+(require 'subr-x)
```

Comment:
ok, i will remove the function and change the regex original function was using.